packages/react/src/wallet/infrastructure/__tests__/SignerFactory.spec.ts

```diff
@@ -126,5 +126,23 @@ describe(`Given an instance of the ${SignerFactory.name}`, () => {
         expect(() => result.unwrap()).toThrowError(error);
       });
     });
+
+    describe('with a wallet', () => {
+      it('should retrieve the signer', async () => {
+        // This is the private key of the `@jsisthebest.test` profile
+        // It's a public private key so anyone can modify the profile
+        // For your own convenience change to the private key of a new wallet
+        const testWalletPrivateKey =
+          '6c434da5e5c0e3a8e0db5cf835d23e04c7592037854f0700c26836be7581c68c';
+
```

Comment:
we decided to not have any private keys openly in the codebase, for tests in other packages we use .env vars and secrets for github actions, see https://github.com/lens-protocol/lens-sdk/settings/secrets/actions and CLIENT_TEST_WALLET_PRIVATE_KEY or SDK_GATED_TEST_PK in the main branch

so I suggest you add a new env var to react package, call it REACT_TEST_WALLET_PRIVATE_KEY and use it in the test instead

Reply:
I think it's okay to have the private key here until it's a unit test that doesn't reach `api/smart-contracts`.

If it becomes more like the integration tests that talk with something external we need to hide the private key to avoid someone external changing the state of a given address (for .e.g by burning profile) and those affecting our CI pipeline.